static/js/util/redux.js

```diff
@@ -0,0 +1,25 @@
+// @flow
+import type {
+  ActionCreator,
+  AsyncActionHelper,
+  AsyncActionCreator
+} from '../flow/reduxTypes';
+import type { Dispatch } from 'redux';
+
+export function createActionHelper(dispatch: Dispatch, actionCreator: Function): (...args: any) => void {
+  return (...args) => dispatch(actionCreator(...args));
+}
+
+export type ActionHelpers = Array<{[k: string]: (...args: any) => void}>;
+export function createSimpleActionHelpers(dispatch: Dispatch, actionManifest: ActionCreator[]): ActionHelpers {
+  return actionManifest.map(actionCreator => (
+    { [actionCreator.name]: createActionHelper(dispatch, actionCreator) }
+  ));
+}
+
+export type AsyncActionHelpers = Array<{[k: string]: AsyncActionHelper}>;
+export function createAsyncActionHelpers(dispatch: Dispatch, actionManifest: AsyncActionCreator[]): AsyncActionHelpers {
+  return actionManifest.map(actionCreator => (
+    { [actionCreator.name]: createActionHelper(dispatch, actionCreator) }
+  ));
+}
```

Comment:
wait... these are the same function `createSimpleActionHelpers ` and `createAsyncActionHelpers`. i wondered this before but forgot about it: why not just return the promise from `dispatch` in all cases? if the client doesn't do anything with the return value, that's on them. don't think it has any performance penalty. then we could combine these two into one called `createActionHelpers`.

also, i think we should rename `actionManifest` to `actions` or `actionList`